soroban-sdk/src/token.rs

```diff
@@ -288,6 +288,33 @@ pub struct Clawback {
     pub amount: i128,
 }
 
+/// This is a helper function to publish either a `Transfer` or `TransferMuxed`
+/// event based on whether the `to` address is a muxed address or not (i.e.
+/// whether it has non-None ID).
+pub fn publish_transfer_to_muxed_address_event(
```

Comment:
I think the name can be shortened and still retain the same meaning.
```suggestion
pub fn publish_transfer_event(
```

Reply:
Renamed.